pipeline/pipeline-notes.rst

```diff
@@ -14,47 +14,105 @@ and will then compile the paper from scratch using the new figures.
 Starting up a machine and installing software
 ---------------------------------------------
 
-First, start up an EC2 instance with the AWS console. Be sure to choose an
+If you are going to run this pipeline on a cloud machine, we recommend Amazon EC2 or RackSpace.
+These notes will give specific instructions to both of those machine types.
+
+EC2
+--------------------------------------------
+When starting an EC2 instance with the AWS console. Be sure to choose an
 Ubuntu 14.04 instance. The c3.2xlarge instance size will work just fine, but
 do not choose a smaller instance than this.
 
 Make sure you edit your security groups to include port 22 (SSH) and port 
-80 (HTTP) ; you'll need the first one to log in, and the second one to 
+8080 ; you'll need the first one to log in, and the second one to 
 connect to the ipython notebook.
 
-Once you ssh in, set yourself up to run as root.::
+If you need to create a new user, you can do so with::
 
- sudo su
+  sudo useradd -m -d /home/user <your_user_name>
 
-Next, we are going to set the instance up with many of the software 
-packages we will need::
+We will also want to link /mnt/bin directory to our home/bin directory::
+  sudo ln -s /mnt/bin ${HOME}/bin
```

Comment:
When EC2 is available again, please test this out. Where does `/mnt/bin/` come from?